mne/io/eeglab/eeglab.py

```diff
@@ -576,19 +585,74 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
         logger.info('Ready.')
 
 
-def _read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer'):
-    """Create events array from EEGLAB structure.
+def read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer',
+                       uint16_codec=None):
+    r"""Create events array from EEGLAB structure.
```

Comment:
To be honest I put it there because I saw it in the other docstrings. I can remove it, but it's more consistent with the others in that file this way ...